addons/hr_holidays/models/hr_holidays.py

```diff
@@ -372,6 +376,20 @@ def unlink(self):
     ####################################################
 
     @api.multi
+    def subscribe_followers(self):
```

Comment:
Now that we effectively change state, couldn't we move that into the _track_subtype method to use the automatic traking of mail ?